guillotina/contrib/dbusers/groups.py

```diff
@@ -0,0 +1,20 @@
+from guillotina import configure
+from guillotina.auth.groups import GuillotinaGroup
+from guillotina.interfaces import IGroups
+
+
+@configure.utility(provides=IGroups)
+class GroupsUtility:
+    """ Class used to get groups. """
+
+    def get_principal(self, ident, principal=None):
+        # import pdb; pdb.set_trace()
```

Comment:
you probably need some db lookups here right?

something like
```python
try:
    group = await navigate_to(get_current_container(), f"groups/{ident}")
except KeyError:
    return None ?
return GuillotinaGroup(
    ident,
    roles=group.roles,
    permissions=group.permissions
)
```

Reply:
Exactly this, but I need to make get_principals async, and I'm not sure this is a change we can make